app/models/assets_model.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+class assets_model extends Model {
+    private function execute($sql, $datatype = null, $data = array()) {
+        $stmt = $this->DB->prepare($sql);
+        if (!$stmt) {
+            return false;
+        }
+        if (!empty($datatype) && !empty($data)) {
+            if (!call_user_func_array(array($stmt, "bind_param"), array_merge([$datatype], $data))) {
+                return false;
+            }
+        }
+        if (!$stmt->execute()) {
+            return false;
+        }
+        $result = $stmt->get_result();
+        if ($result) {
+            return $result;
+        }
+        return $stmt;
+    }
+
+    #TODO Duplication of Assets
```

Comment:
I like this approach of mentioning todos. This way it will be easier to see all the todo with a simple grep. :+1: +100 for this method of mentioning todos.